src/app/monitoring/monitoring.component.ts

```diff
@@ -121,8 +122,27 @@ export class MonitoringComponent implements OnInit {
     this.store.dispatch(new LoadBindings());
     this.store.dispatch(new LoadPanels());
     this.loadPanels();
+    // end panel edit mode when other sidebar section gets clicked
+    this.store.select(getState).pipe(filter(k => this.shouldEndPanelEditMode(k))).subscribe((state: any) => {
+        this.editModeListener(); // turning off panel edit mode
+    })
   }
 
+  // this method expects 'Panels' section to be the first section in the sideBar and 'add Panel' to be the last item in the Panels section
+  shouldEndPanelEditMode(state: any): boolean {
+    if (!this.panelEditMode){ return false }
```

Comment:
this is the very first thing that has to be done in the filter. Why shopuld we use a method for this.

Reply:
changed and pushed it